src/aws-api-mcp-server/README.md

```diff
@@ -205,8 +205,8 @@ See [DEPLOYMENT.md](https://github.com/awslabs/mcp/blob/main/src/aws-api-mcp-ser
-| `AWS_API_MCP_WORKING_DIR`                                         | ❌ No                       | \<Platform-specific temp directory\>/aws-api-mcp/workdir | Working directory path for the MCP server operations. Must be an absolute path when provided. Used to resolve relative paths in commands like `aws s3 cp`. Does not provide any sandboxing or security restrictions. If not provided, defaults to a platform-specific directory:<br/><br/>• **Windows**: `%TEMP%\aws-api-mcp\workdir` (typically `C:\Users\<username>\AppData\Local\Temp\aws-api-mcp\workdir`)<br/>• **macOS**: `/private/var/folders/<hash>/T/aws-api-mcp/workdir`<br/>• **Linux**: `$XDG_RUNTIME_DIR/aws-api-mcp/workdir` (if set) or `$TMPDIR/aws-api-mcp/workdir` (if set) or `/tmp/aws-api-mcp/workdir` |
-| `AWS_API_MCP_ALLOW_UNRESTRICTED_LOCAL_FILE_ACCESS`                | ❌ No                       | `"workdir"`                                              | Controls file system access level with three modes:<br/><br/>• `"unrestricted"`: Enables system-wide file access (may cause unintended overwrites)<br/>• `"workdir"`: Restricts file operations to `AWS_API_MCP_WORKING_DIR` (default)<br/>• `"no-access"`: Rejects all local file path arguments in AWS CLI commands and blocks commands requiring local file access (e.g., `aws cloudformation package`, `aws ecs deploy`). S3 URIs (`s3://...`) and stdout redirect (`-`) remain allowed.<br/><br/>**DEPRECATED**: The boolean values `"true"` and `"false"` are supported for backward compatibility. Use `"unrestricted"` instead of `"true"` and `"workdir"` instead of `"false"`. |
+| `AWS_API_MCP_WORKING_DIR`                                         | ❌ No                       | \<Platform-specific temp directory\>/aws-api-mcp/workdir | Working directory path for the MCP server operations. Must be an absolute path when provided. Used to resolve relative paths in commands like `aws s3 cp`. When file access mode is `"workdir"` (default), file operations are restricted to this directory. Set a broad path for your use case when using "workdir" mode. If not provided, defaults to a platform-specific directory:<br/><br/>• **Windows**: `%TEMP%\aws-api-mcp\workdir` (typically `C:\Users\<username>\AppData\Local\Temp\aws-api-mcp\workdir`)<br/>• **macOS**: `/private/var/folders/<hash>/T/aws-api-mcp/workdir`<br/>• **Linux**: `$XDG_RUNTIME_DIR/aws-api-mcp/workdir` (if set) or `$TMPDIR/aws-api-mcp/workdir` (if set) or `/tmp/aws-api-mcp/workdir` |
```

Comment:
Why are we suggesting broad path? This might have security implications

Reply:
If ambiguity exists, expect it to be interpreted in the least safe way. Think of Murphy’s Law.. user's might still end up shooting themselves in the foot when using this, but we should do our best to prevent this. Instead of suggesting broad paths, we should have more defensive stance, and rather say something along the lines of "pick a dir where you feel safe allowing agent full read & write access". That's why the original "Does not provide any sandboxing or security restrictions" was in there.